src/Facades/JWTAuth.php

```diff
@@ -11,8 +11,30 @@
 
 namespace Tymon\JWTAuth\Facades;
 
+use Tymon\JWTAuth\Token;
+use Tymon\JWTAuth\Manager;
+use Tymon\JWTAuth\Payload;
+use Tymon\JWTAuth\Blacklist;
+use Tymon\JWTAuth\Claims\Factory;
 use Illuminate\Support\Facades\Facade;
+use Tymon\JWTAuth\Contracts\Providers\JWT as Provider;
```

Comment:
are these imports necessary?

Reply:
Well, no, not technically. They're standard for readability.